junit5-jupiter-starter-ant/build.sh

```diff
@@ -3,7 +3,7 @@
 #
 # Set constants.
 #
-junit_platform_version='1.5.2'
+ivy_version='2.5.0-rc1'
```

Comment:
❓ Does it have to be an RC or can we use a stable version instead?

Reply:
Indeed. No problem at stable version.